independent-projects/bootstrap/maven-plugin/src/main/java/io/quarkus/maven/ExtensionDescriptorMojo.java

```diff
@@ -153,11 +158,41 @@ public void execute() throws MojoExecutionException {
         if(extObject.get("description") == null && project.getDescription() != null) {
             extObject.add("description", project.getDescription());
         }
-
+        
         try (BufferedWriter bw = Files.newBufferedWriter(output.resolve(BootstrapConstants.EXTENSION_PROPS_JSON_FILE_NAME))) {
             extObject.writeTo(bw, WriterConfig.PRETTY_PRINT);
         } catch (IOException e) {
             throw new MojoExecutionException("Failed to persist " + output.resolve(BootstrapConstants.EXTENSION_PROPS_JSON_FILE_NAME), e);
         }
     }
+
+	private void transformLegacyToNew(final Path output, JsonObject extObject) throws MojoExecutionException {
+		JsonObject metadata = new JsonObject();
```

Comment:
No need to create a new here, better do that in an `else` below

Reply:
fixed.